recipes/openssl/1.x.x/conandata.yml

```diff
@@ -26,6 +26,11 @@ sources:
       - "https://www.openssl.org/source/openssl-1.1.1t.tar.gz"
       - "https://www.openssl.org/source/old/1.1.1/openssl-1.1.1t.tar.gz"
       - "https://github.com/openssl/openssl/releases/download/OpenSSL_1_1_1t/openssl-1.1.1t.tar.gz"
+  1.1.1u:
+    sha256: e2f8d84b523eecd06c7be7626830370300fbcc15386bf5142d72758f6963ebc6
+    url:
+      - "https://www.openssl.org/source/openssl-1.1.1u.tar.gz"
```

Comment:
```suggestion
      - "https://www.openssl.org/source/openssl-1.1.1u.tar.gz"
      - "https://www.openssl.org/source/old/1.1.1/openssl-1.1.1u.tar.gz"
```

Reply:
@Croydon, I had tried it with that URL, but it gives me a 404:
```
wget https://www.openssl.org/source/old/1.1.1/openssl-1.1.1u.tar.gz
--2023-05-30 13:41:57--  https://www.openssl.org/source/old/1.1.1/openssl-1.1.1u.tar.gz
Resolving www.openssl.org (www.openssl.org)... 96.16.16.41
Connecting to www.openssl.org (www.openssl.org)|96.16.16.41|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-05-30 13:41:59 ERROR 404: Not Found.
```

Reply:
Looking at [this](https://www.openssl.org/source/old/1.1.1/) URL, I can see that `t` and `u` are not (yet) in the old folder.
If the idea is the futureproof this so that these mirrors will eventually work, I am happy to add them back in the recipe...

Reply:
@gegles That is the idea, yes